static/templates/subscription_settings.handlebars

```diff
@@ -94,8 +95,11 @@
 
         {{#render_subscribers}}
         <div class="subscriber_list_settings">
-            <div class="sub_settings_title float-left">{{t "Stream membership" }}</div>
-            <div class="subscriber_list_add float-right">
+            <div class="sub_settings_title float-left">
+                {{t "Stream membership" }}
+                <div class="stream_subscription_info small"></div>
```

Comment:
I don't understand why this was added, but it seems harmless, so I'm merging.

Reply:
Actually, not merging yet.  The changes here seem somewhat unrelated to the main point of this PR.

Reply:
You are right, this changes are not required for this PR. These changes are covered in PR #7942 for error handling of `Stream membership` section.